res/css/views/context_menus/_NewDropdownMenu.scss

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
Naming nit, any chance we could rename the old one to Legacy and name the new one the final name to help aid future PRs clashing with the future PR which will rename strip the `New` prefix?